eng/Signing.props

```diff
@@ -42,6 +42,21 @@
       <CoreCLRCrossTargetItemsToSign Include="$(CoreCLRArtifactsPath)$(CoreCLRCrossTargetComponentDirName)/sharedFramework/*.exe" />
     </ItemGroup>
 
+    <!-- Ensure that we don't miss the DAC or DBI with the globbing below -->
+    <PropertyGroup Condition="'$(SignDiagnostics)' == 'true'">
```

Comment:
I’ve already validated that these changes work and sign the files correctly. This can replace the SignBinaries group in mmitche’s PR.

Reply:
Can this work outside a target though?

Reply:
It could if L55 would use the WithMetadataValue item function to check on filename but I doubt it's worth changing it now that Jeremy already did the validation.

Reply:
As discussed during standup, since we're just using globs and not using the version variables that Arcade updates, we can use this inside of an InitialTargets target if need be to ensure that we correctly sign the short name DAC, long name DAC, and DBI.